### PR TITLE
[Feature] Prettier code format

### PR DESCRIPTION
**Pull request by flx-sta**
## Why am I making these changes?

We need a code-format(ter). 
it's way overdue!

## What are the changes from a developer perspective?

I've applied prettier with a `printWidth` of `120`.
All other available options can be found [here](https://prettier.io/docs/en/options).

> [!WARNING]
> *Some es-lint rules were screaming at me.*
> 
> One was about `pokerogue/src/battle-scene.ts`
> `2829:11  error  'pokemonModifierChance' is assigned a value but never used  @typescript-eslint/no-unused-vars`.
> 
> I've out-commented the code with a `// TODO: What is the purpose of this? https://discord.com/channels/1125469663833370665/1250836282926436413/1283919020621299772`

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)

### CODE REVIEW

**Review by DayKev — APPROVED** [review]

File: src/configs/inputs/configHandler.ts [hunk 1]
```diff
@@ -115,7 +115,10 @@ export function getIconForLatestInput(configs, source, devices, settingName) {
 
 export function assign(config, settingNameTarget, keycode): boolean {
   // first, we need to check if this keycode is already used on another settingName
-  if (!canIAssignThisKey(config, getKeyWithKeycode(config, keycode)) || !canIOverrideThisSetting(config, settingNameTarget)) {
+  if (
+    !canIAssignThisKey(config, getKeyWithKeycode(config, keycode)) ||
+    !canIOverrideThisSetting(config, settingNameTarget)
```
[the remark this answers is not in the document]
Comment (flx-sta) [reply src/configs/inputs/configHandler.ts:120]: That's legacy code. I ain't touching that in such a PR :D

File: src/data/daily-run.ts [hunk 2]
```diff
@@ -13,62 +18,90 @@ export interface DailyRunConfig {
...
+function getDailyRunStarter(
+  scene: BattleScene,
+  starterSpeciesForm: PokemonSpeciesForm,
+  startingLevel: integer,
+): Starter {
+  const starterSpecies =
+    starterSpeciesForm instanceof PokemonSpecies ? starterSpeciesForm : getPokemonSpecies(starterSpeciesForm.speciesId);
   const formIndex = starterSpeciesForm instanceof PokemonSpecies ? undefined : starterSpeciesForm.formIndex;
-  const pokemon = new PlayerPokemon(scene, starterSpecies, startingLevel, undefined, formIndex, undefined, undefined, undefined, undefined, undefined, undefined);
+  const pokemon = new PlayerPokemon(
```
[the remark this answers is not in the document]
Comment (flx-sta) [reply src/data/daily-run.ts:85]: not really. You would have to set the type up in a way that undefined is not allowed.
But why shouldn't it be there?

---

<sub>Btw any change would be out of scope of this PR</sub>

**Comment by flx-sta** [comment]
Closing for now. As this will probably take ages to make a decision about this.

<sub>Politics</sub>

File: src/data/daily-run.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (DayKev) [reply src/data/daily-run.ts:85]: Well there's no reason to pass undefined to optional parameters, they're optional for a reason. Everything after `formIndex` is unnecessary. Granted, it's a relic from before those fields were made optional.

**Pull request closed without merging**